cmd/as/main.go

```diff
@@ -12,19 +12,31 @@ import (
 )
 
 func main() {
-	store := mustStore()
-	h := server.BuildASRouter(server.Deps{Store: store}, server.Options{})
+	grantStore := mustGrantStore()
+	rsKeyStore := mustRSKeyStore()
+	h := server.BuildASRouter(server.Deps{
+		GrantStore: grantStore,
+		RSKeyStore: rsKeyStore,
+	}, server.Options{})
 	log.Fatal(http.ListenAndServe(":8085", h))
 }
 
-func mustStore() types.Store {
+func mustGrantStore() types.GrantStore {
 	s, err := gnap.NewFileStore(defaultDataDir(), types.Config{GrantTTLSeconds: 120})
```

Comment:
This is where you would increase the grant store expiration, for longer testing.